bootstrap/src/lib/cache/index.ts

```diff
@@ -70,10 +70,9 @@ export const redactOptions = (options: CacheOptions) => ({
       : local.redactOptions(options.cacheOptions),
 })
 
-export const withCache = async (
-  execute: ExecuteWrappedResponse,
-  options: CacheOptions = defaultOptions(),
-) => {
+export const withCache = async (execute: Execute): Promise<Execute> => {
+  const options = defaultOptions()
```

Comment:
Why did this API change? Shouldn't anyone using the cache be able to configure the cache?

Reply:
Yes they should. I was having a problem with the Middleware type, this shouldn't be changed. Reverted.